libs/barista-components/filter-field/README.md

```diff
@@ -27,6 +27,9 @@ class MyModule {}
 | `disabled`   | `boolean`                 | `false` | Whether the filter-field is disabled.                                                                            |
 | `aria-label` | `string`                  |         | Sets the value for the Aria-Label attribute.                                                                     |
 
+_Note: Checkout the `Data source` section to understand how to provide the
```

Comment:
Can we make the Data source section into an anchor link to that section. Would probably be helpful